net/wfb-ng/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wfb-ng
+PKG_VERSION:=24.9.1
```

Comment:
This version is not released nor tagged.

Reply:
https://github.com/svpcom/wfb-ng/tree/wfb-ng-24.09-openwrt-preview
I've added tag for it

Reply:
I have a lot of requests from wfb-ng users that want to try wfb-ng on their wifi routers but for most of them compiling external packets is too hard that is why I've done this pull-request